app/components/dashboard/collection_component.html.erb

```diff
@@ -28,6 +34,12 @@
                 <span class="fas fa-pencil-alt"></span>
               <% end %>
             <% end %>
+
+            <% if allowed_to?(:delete?, work) %>
+              <%= link_to work_path(work), method: :delete, aria: { label: "Delete #{work.title}"}, data: {confirm: "Are you sure you want to delete this draft work? It cannot be undone."} do %>
```

Comment:
Please add spaces around the hash brackets.

Reply:
Time to consider erb-lint?